portfolio/src/main/webapp/script.js

```diff
@@ -419,6 +422,36 @@ async function deleteAndFetchEmpty() {
     document.getElementById("form-container").innerText = value;
 }
 
+//*********************** AUTHENTICATION ***********************
+/**
+ * Allows for authentication
+ */ 
+async function displayAuth(){
+    const response = await fetch('/auth');
+    const value = await response.json();
```

Comment:
nit: name this `value` `authenticationState` or something similar. Otherwise this code is vague.